### ISSUE TITLE
Outdated BusyBox 

### ISSUE THREAD

**Issue by eastcoastcoder** [SVoxel/R9000#2]
I’ve noticed the version of BusyBox included is 1.4.2 from 2007. I’m having issues getting some modern ssh software working correctly with this.
I have attempted to upgrade from opkg, even adjusted my path and set an alias to /opt/bin/busybox but it still gives me 1.4.2 when I try to use it with this ssh software.

Is there any way this could include a new version baked in?

**Comment by SVoxel** [comment]
There are some specific features of this old BusyBox. It is changed by Netgear's outsource company DNI (Delta Network Inc). And just upgrade if BB could lead to a lot of scripts failed in my firmware. Just because of these scripts are not using anything modern. E.g. differences could be  in tail, head, cat grep etc. BusyBox is second or even third importance for me. There will be headache to integrate the changes from next stock release modifying all the scripts to be workable with new BB I use.

So if you want to have modern BB just go with Entware. Install alternative version of Entware instead of generic. See the difference:

https://github.com/Entware/Entware/wiki/Alternative-install-vs-standard

The link on my site for alternative Entware initial installation:

https://www.voxel-firmware.com/Downloads/Voxel/Entware/entware-cortex-a15-3x-initial-alternative.tar.gz

BB version in Entware is 1.30. Also you can even install core utils from Entware instead of using BB.

Voxel.

**Comment by eastcoastcoder** [comment]
Yeah I was afraid there are some optimizations to the BusyBox which might make this request unfeasible so I totally understand.
So I did install the Entware version and it works fine in an interactive terminal but there's some missing piece still. The way the software I'm trying to use works is that it fires off single commands at a time without an interactive terminal. As a result, it seems like any Entware are falling back to their builtin versions.
<img width="572" alt="BusyBox Entware" src="https://user-images.githubusercontent.com/4449468/60739085-84fd1000-9f2e-11e9-9d17-2f90f3b7fedb.png">
<img width="574" alt="BusyBox SSH" src="https://user-images.githubusercontent.com/4449468/60739087-84fd1000-9f2e-11e9-9fad-a05629c7e216.png">
<img width="569" alt="Find Entware" src="https://user-images.githubusercontent.com/4449468/60739089-8595a680-9f2e-11e9-92c9-f1f1e1327365.png">
<img width="576" alt="Find SSH" src="https://user-images.githubusercontent.com/4449468/60739090-8595a680-9f2e-11e9-9a09-ae61389d7bf0.png">
